pom.xml

```diff
@@ -1456,6 +1456,8 @@
     <profile>
       <id>templates</id>
       <modules>
+        <module>cdap-api</module>
```

Comment:
the cdap-api and cdap-unit-test are in the top level module section. They are always included. For not using snapshot version from `mvn install`, just make sure the version we set for the dependency is `${project.version}`.